snap/hooks/install

```diff
@@ -42,24 +42,24 @@ if [ ! -f "$SNAP_DATA/config/security-secretstore-setup/res/kong-admin-config.te
         "$SNAP_DATA/config/security-secretstore-setup/res/kong-admin-config.template.yml"
 fi
 
-# handle app-service-configurable's various profiles:
-# 1. ensure all the directories from app-service-configurable exist
+# handle app-service-configurable's rules-engine profile
+# 1. create the directory
 # 2. copy the config files from $SNAP into $SNAP_DATA
-# 3. replace the various env vars that might be in that config file with their
-#    "current" symlink equivalent
 mkdir -p "$SNAP_DATA/config/app-service-configurable/res/rules-engine"
-RULES_ENGINE_PROFILE_CONFIG="config/app-service-configurable/res/rules-engine/configuration.toml"
 if [ ! -f "$SNAP_DATA/$RULES_ENGINE_PROFILE_CONFIG" ]; then
-    cp "$SNAP/$RULES_ENGINE_PROFILE_CONFIG" "$SNAP_DATA/$RULES_ENGINE_PROFILE_CONFIG"
-    sed -i -e "s@\$SNAP_COMMON@$SNAP_COMMON@g" \
-        -e "s@\$SNAP_DATA@$SNAP_DATA_CURRENT@g" \
-        -e "s@\$SNAP@$SNAP_CURRENT@g" \
-        "$SNAP_DATA/$RULES_ENGINE_PROFILE_CONFIG"
+    cp "$SNAP/config/res/rules-engine/configuration.toml" \
```

Comment:
Where is $RULES_ENGINE_PROFILE_CONFIG set and should we be checking for that rather than directly for "$SNAP_DATA/config/app-service-configurable/res/rules-engine/configuration.toml ?

Reply:
@siggiskulason is is set via `-profile rules-engine` on line 364 of snapcraft.yaml

Reply:
There isn't a `RULES_ENGINE_PROFILE_CONFIG` that I see.